test/serializer/abstract/ObjectAssign4.js

```diff
@@ -1,6 +0,0 @@
-var obj = global.__abstract && global.__makePartial ? __makePartial(__abstract({}, "obj")) : {};
-var copyOfObj = Object.assign({}, obj);
-
-inspect = function() {
-  return copyOfObj;
-}
```

Comment:
Why delete this test?

Reply:
There was some confusion over the scope of this PR as the other PR blurred it all.